common/djangoapps/student/roles.py

```diff
@@ -245,6 +245,33 @@ class OrgRole(RoleBase):
     def __repr__(self):
         return f'<{self.__class__.__name__}>'
 
+    def update_org_course_role(self, user, orgs):
+        """
+        Updates(creates/deletes) the org role specified for the user
+        for multiple organizations.
+
+        Args:
+            user: User object
+            orgs: List of organization names
+        """
+        from common.djangoapps.student.models import CourseAccessRole  # lint-amnesty, pylint: disable=redefined-outer-name, reimported
+        org_role_to_delete = []
+        orgs_role_to_create = []
+        existing_org_role = set(CourseAccessRole.objects.filter(user=user, role=self._role_name)
```

Comment:
```suggestion
        existing_org_roles = set(CourseAccessRole.objects.filter(user=user, role=self._role_name)
```

Reply:
IMHO if the org roles are empty, it would not even reach the `if org_role_to_delete` block since it will be an empty set, so I think this will be a redundant check.